Maquetado/scss/styles.scss

```diff
@@ -1,10 +1,58 @@
 @import 'colors';
 @import 'site-sizes';
 
-body{
+body {
   font-family: 'Open Sans', sans-serif;
 
   .container {
+    align-items: center;
+    display: grid;
+    grid-template-areas:
+      "front-title  front-title"
+      "main-front   front"
+      "back-title   back-title"
+      "main-back    back";
+
+    grid-template-columns: 40% 37%;
+    grid-template-rows: 96px 864px 96px 864px;
     padding: 128px 48px 48px;
+
+    .inner-grid {
+      align-items: center;
+      border-left: 2px $green solid;
+      display: grid;
+      height: 100%;
+      grid-template-areas:
```

Comment:
maybe for this case is better to define a grid with grid-template-rows and grid-template-columns. Then you can set a special width or height depending on how many columns/rows the component needs.